azure-pipelines.yml

```diff
@@ -2,11 +2,26 @@
 # Start with a minimal pipeline that you can customize to build and deploy your code.
 # Add steps that build, run tests, deploy, and more:
 # https://aka.ms/yaml
+#
+# | Type               | Master | Beta | Release | LTS  | Tags |
+# | ---                | ---    | ---  | ---     | ---  | ---  |
+# | Lint               | yes    | yes  | no      | no   | no   |
+# | Basic Tests        | yes    | yes  | yes     | yes  | no   |
+# | Build  Tests       | yes    | yes  | yes*    | yes* | no   |
+# | Windows Tests      | yes    | yes  | yes*    | yes* | no   |
+# | Floating Deps      | yes    | yes  | yes     | yes  | no   |
+# | Ember LTS Branches | yes    | yes  | yes     | yes  | no   |
+# | Release Channels   | yes    | yes  | no      | no   | no   |
+# | External Partners  | yes    | yes  | no      | no   | no   |
+#
+# * Runs but skips tests for in-progress-features
 
 trigger:
   - master
   - beta
   - release
+  - release-*
+  - lts-*
```

Comment:
unsure if Azure has the same issue but in Travis for some PRs we had to also have the org name for the repo specified e.g. `emberjs:release-*`